### PR TITLE
Remove positionCount check in createRandomLongDecimalsBlock

### PR DESCRIPTION
**Pull request by yingsu00**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java [hunk 1]
```diff
@@ -265,6 +265,7 @@ public static Block createBooleansBlock(Iterable<Boolean> values)
         return builder.build();
     }
 
+    // It is recommended to have positionCount >= 10 when nullRate > 0, so that the nullRate can be close to the desired value.
```
Comment (mbasmanova) [inline presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java:268]: @yingsu00 

nit: consider shortening to `Use positionCount >= 10 when nullRate > 0...`

What does `nullRate can be close to the desired value` mean?

Comment (yingsu00) [reply presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java:268]: @mbasmanova thanks for the suggestion. By `nullRate can be close to the desired value`, I meant that the actual null rate would be more accurate(close to the specified nullRate) when positionCount is larger.

Comment (mbasmanova) [reply presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java:268]: @yingsu00 Maybe write it like `Make sure positionCount is sufficiently large if nullRate is small`.

Comment (yingsu00) [reply presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java]: Thanks @mbasmanova. I updated the comment to be 
```
// Note: Make sure positionCount is sufficiently large if nullRate is greater than 0
```

File: presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java [hunk 2]
```diff
@@ -708,15 +713,18 @@ public static DictionaryBlock createRandomDictionaryBlock(Block dictionary, int
...
         if (isView) {
             idsOffset = min(ThreadLocalRandom.current().nextInt(dictionary.getPositionCount()), 1);
         }
 
-        int[] ids = IntStream.range(0, positionCount + idsOffset).map(i -> ThreadLocalRandom.current().nextInt(dictionary.getPositionCount() / 10)).toArray();
+        int[] ids = IntStream.range(0, positionCount + idsOffset).map(i -> ThreadLocalRandom.current().nextInt(max(dictionary.getPositionCount() / 10, 1))).toArray();
         return new DictionaryBlock(idsOffset, positionCount, dictionary, ids, false, randomDictionaryId());
     }
 
+    // It is recommended to have positionCount >= (10 / nestedNullRate) when nestedNullRate > 0, so that the nullRate can be close to the desired value.
```
Comment (mbasmanova) [inline presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java:727]: @yingsu00 should this comment be updated to match the rest?

Comment (yingsu00) [reply presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java]: @mbasmanova   Sorry missed this one. I have updated it to be
```
// Note: Make sure positionCount is sufficiently large if nestedNullRate or primitiveNullRate is greater than 0
public static Block createRandomBlockForType(
```

**Pull request merged**